### PR TITLE
AR-37 -> dev

### PR DESCRIPTION
**Pull request by Beafowl-Pull**
## Describe your changes
- Add the deployment on server
- Add the release handler with automatic tag with commit
- Init the backend
## Issue ticket number and link
- https://bravmm.atlassian.net/browse/AR-37
- https://bravmm.atlassian.net/browse/AR-6
- https://bravmm.atlassian.net/browse/AR-7
## Checklist before requesting a review
- [x] I have performed a self-review of my code
- [ ] I have documented my code
- [ ] Do we need to implement unit tests?

### CODE REVIEW

File: back/index.js
Comment (badidu09) [inline back/index.js:1]: Should be in ts ?

**Review by badidu09 — CHANGES_REQUESTED** [review]
Change index.js to index.ts

**Review by badidu09 — APPROVED** [review]

**Review by Vivelis — APPROVED** [review]
LGTM

File: back/index.tsx [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+    res.json({ message: "Welcome to my application." })
+})
+
+// port
+
+const PORT = 8080
+
+app.listen(PORT, () => {
+    console.log(`Server is running on port ${PORT}`)
+})
```
Comment (romainpanno) [inline back/index.tsx:31]: Eof

File: back/.gitignore [hunk 2]
```diff
@@ -1,130 +1,156 @@
...
+/packages/**/*.xcframework
+/packages/**/*.spec.json
+/packages/**/Info-generated.plist
+!crsqlite.xcframework
+
+# iOS
+/ios/Pods
+/ios/build
+**/ios/.xcode.env.local
+/ios/versioned-react-native/*/ReactNative/sdks/hermes-engine/destroot
```
Comment (romainpanno) [inline back/.gitignore:156]: Eof rien à foutre on tiens la norme

File: .github/workflows/deploy-on-server.yml [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+      - name: executing remote ssh commands using ssh key
+        if: github.ref == 'refs/heads/dev'
+        uses: appleboy/ssh-action@v1.0.0
+        with:
+          host: ${{ secrets.FTP_SERVER }}
+          username: root
+          password: ${{ secrets.FTP_PASSWORD }}
+          key: ${{ secrets.SSH_KEY }}
+          port: 2121
+          script: reboot
```
Comment (romainpanno) [inline .github/workflows/deploy-on-server.yml:57]: Eof

File: .github/workflows/create-apk.yml [hunk 4]
```diff
@@ -12,14 +12,20 @@ jobs:
...
-          expo-version: latest
+          eas-version: latest
           token: ${{ secrets.EXPO_CLI_TOKEN }}
 
       - name: 🔧 execute expo build
+        working-directory: ./front
         timeout-minutes: 30
         run: |
-          cd front
           eas build --non-interactive --platform android --profile development
```
Comment (romainpanno) [inline .github/workflows/create-apk.yml:31]: Eof

**Review by romainpanno — CHANGES_REQUESTED** [review]
Eof

**Review by romainpanno — APPROVED** [review]

**Review by Vivelis — APPROVED** [review]

**Review by badidu09 — APPROVED** [review]

**Pull request merged**